pkg/apis/eksctl.io/v1alpha5/assets/schema.json

```diff
@@ -1151,14 +1173,20 @@
       "properties": {
         "outputPath": {
           "type": "string",
+          "description": "Output directory for the processed profile templates (generate profile command) Defaults to ./<quickstart-repo-name>",
+          "x-intellij-html-description": "Output directory for the processed profile templates (generate profile command) Defaults to ./<quickstart-repo-name>",
```

Comment:
sorry didn't see this, it gets eaten as an HTML tag

Reply:
fixed